### ISSUE TITLE
browse interface to metacat

### ISSUE THREAD

**Issue by mbjones** [NCEAS/metacat#164]
---

Author Name: **Matt Jones** (Matt Jones)
Original Redmine Issue: 433, https://projects.ecoinformatics.org/ecoinfo/issues/433
Original Date: 2002-02-19
Original Assignee: Matt Jones

---

Searching in metacat works fine, but as the number of enries grows we are going
to want to be able to browse through the entries in some sort of organized way.
 Here's a proposed approach.

Create a controlled vocabulary consisting of the terms that we want to use for
browsing.  Next, classify the data sets using this controlled vocabulary.
Finally, create an inverse index of the datasets using the categories, such that
each dataset that is classified using a particular term is listed under that
term.  Any data set can show up under multiple terms.  Data sets that are not
classified using that controlled vocabulary would be omitted from the list.

This system would allow multiple classifications to coexist.  One could switch
browse views by simply switching which controlled vocabulary was being used
(e.g., switch from a taxonomically oriented classification to a geologically
oriented classification).  The initial classification could simply be to use
uncontrolled keywords (e.g., no thesaurus), thus it would likely have almost as
many categories as there are data sets, but it wold give some sense of the use
of keywords.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Matt Jones** (Matt Jones)
Original Date: 2004-09-21T11:08:36Z

---

I began this work, and now have some XSLT stylesheets that allow you to browse
via keyword, organizaiton, and creator.  However, these didn't prove to be that
useful, so they were left in the development skin.  Instead, we developed a
simple categorization and allowed keyword searches based on those categories. 
See the KNB site.  May revisit this later, but for now it'll remain in POSTPONE.
